metagraph/tests/graph/all/test_dbg_helpers.cpp

```diff
@@ -1,6 +1,10 @@
 #include "test_dbg_helpers.hpp"
 
+#include "../../annotation/test_annotated_dbg_helpers.hpp"
+#include "annotation/representation/column_compressed/annotate_column_compressed.hpp"
+
 #include "gtest/gtest.h"
+#include "graph/annotated_dbg.hpp"
```

Comment:
Why? The PR doesn't seem to add anything that needs them in this file?

Reply:
Thanks, this made it here accidentally, it should be part of #500 instead